docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`-` Exchange data imports will now warn when duplicate entries are found.
```

Comment:
Is it just exchange or all CSV imports? I think it should be for all CSV imports, right?